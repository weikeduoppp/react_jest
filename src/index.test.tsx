const getUserById = (id: string) => {
  return new Promise((resolve) => {
    resolve({
      data: id,
      code: 0,
    });
  });
};

describe("getUserById", () => {
  it("获取 user === 1 的用户", async () => {
    const res = await getUserById("1");
    expect(res).toMatchSnapshot();
  });
});
