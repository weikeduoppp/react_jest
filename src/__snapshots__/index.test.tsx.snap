// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getUserById 获取 user === 1 的用户 1`] = `
Object {
  "code": 0,
  "data": "1",
}
`;
